core/templates/dev/head/domain/story/EditableStoryBackendApiServiceSpec.js

```diff
@@ -82,6 +82,22 @@ describe('Editable story backend API service', function() {
     }
   );
 
+  it('should successfully delete a story from the backend',
+    function() {
+      var successHandler = jasmine.createSpy('success');
+      var failHandler = jasmine.createSpy('fail');
+
+      $httpBackend.expect('DELETE', '/story_editor_handler/data/0/1').respond(
+        200);
+      EditableStoryBackendApiService.deleteStory('0', '1').then(
```

Comment:
Generally, in tests, might be a better idea to name your args semantically (e.g. 'topicId', 'storyId' or 'topicId0', 'storyId1'). Both this and what you have here are technically "correct", but the former makes it easier to figure out what the args are referring to.

Reply:
Ok, done.